index.bs

```diff
@@ -1400,19 +1412,19 @@ The [=authenticator data=] structure is a byte array of 37 bytes or more, as fol
     </tr>
 </table>
 
-The RP ID is originally received from the client when the credential is created, and again when an assertion is generated.
-However, it differs from other [=client data=] in some important ways. First, unlike the client data, the RP ID of a credential
+The [=RP ID=] is originally received from the client when the credential is created, and again when an assertion is generated.
+However, it differs from other [=client data=] in some important ways. First, unlike the client data, the [=RP ID=] of a credential
 does not change between operations but instead remains the same for the lifetime of that credential. Secondly, it is validated
-by the authenticator during the [=authenticatorGetAssertion=] operation, by verifying that the RP ID associated with the
-requested credential exactly matches the RP ID supplied by the client.
+by the authenticator during the [=authenticatorGetAssertion=] operation, by verifying that the [=RP ID=] associated with the
+requested credential exactly matches the [=RP ID=] supplied by the client.
 
 The `TUP` flag SHALL be set if and only if the authenticator detected a user through an authenticator specific gesture. The `RFU` bits
 SHALL be set to zero.
 
 For attestation signatures, the authenticator MUST set the AT flag and include the [=attestation data=]. For authentication
 signatures, the AT flag MUST NOT be set and the [=attestation data=] MUST NOT be included.
 
-If the authenticator does not include any extension data, it MUST set the `ED` flag in the first byte to zero, and to one if
+If the authenticator does not include any extension data, it MUST set the `ED` flag to zero; it is set to one if
```

Comment:
Ok, done. tho the "..in the first byte.." portion of the original wording is incorrect and i've left it out.